install/ubuntu/install.ubuntu.sh

```
@@ -0,0 +1,86 @@
+    confirmation=$1
+  else
+    read -p "Enter your choice: " confirmation
+  fi
+  if [ "$confirmation" == "install" ]; then
+    if [ -n "$INSTALL_DIR" ] && [ "$INSTALL_DIR" != "" ]; then
+      echo "Removing existing installation..."
+      service nginx stop 2>/dev/null
+      pkill -f "python /app/server" 2>/dev/null
+      umount  "$INSTALL_DIR/api" 2>/dev/null
+      umount  "$INSTALL_DIR/front" 2>/dev/null
+      rm -rf "$INSTALL_DIR/"
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)